content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -192,7 +192,7 @@ To provide access to the secret we created, follow the steps mentioned below:
 
 Figure 7
 
-## **Create a new ASP.NET 5 Core application**
+## Create a new ASP.NET 5 Core application
 
 First of all, you will create an ASP.NET Core 5 web application. The project type comes bundled with all the template files to create a web application, even before you add something. Follow the steps given below to create a new ASP.NET Core Web application within the Visual Studio 2019 IDE.
```

Comment:
Either end points with a pull stop or doesn't use a pull stop. But please make sure usage of pull stop is consistent through the articles wherever points are present. Our recommendation is not to use a pull stop in points.

Reply:
Fixed, thanks!